tests/test_sinogram.py

```diff
@@ -135,4 +139,84 @@ def test_multidim(num_ang):
     _nrms = np.sqrt(np.mean((s - reference_sinograms) ** 2, axis=-1)) / np.linalg.norm(
         reference_sinograms, axis=-1
     )
-    np.testing.assert_array_less(_nrms, SK_TOL, "Error in image projections.")
+    np.testing.assert_array_less(
+        _nrms, SK_TOL_FORWARDPROJECT, "Error in image projections."
+    )
+
+
+def test_back_project_single(masked_image, num_ang):
+    """
+    Test Line.backproject on a single stack of line projections or sinogram. Compares the reconstructed image to original.
+    """
+    angles = np.linspace(0, 360, num_ang, endpoint=False)
+    rads = angles / 180 * np.pi
+    sinogram = masked_image.project(rads)
+    sinogram_np = sinogram.asnumpy()
+    back_project = sinogram.back_project(rads)
+
+    assert masked_image.shape == back_project.shape, "The shape must be the same."
+
+    # generate circular mask w/ radius 1 to reconstructed image
+    # aim to remove discrepencies for the edges of the image
+    g = grid_2d(back_project.resolution, normalized=True, shifted=True)
+    mask = g["r"] < 1
+    our_back_project = back_project.asnumpy()[0] * mask
+
+    # generating sci-kit image backproject method w/ no filter
+    sk_image_iradon = iradon(sinogram_np[0].T, theta=angles[::-1], filter_name=None)
+
+    # we apply a normalized root mean square error on the images to find relative error to range of ref. image
+    # Note: tolerance is typically < 0.2 regardless of angles, pixels, etc.
+    nrmse = np.sqrt(np.mean((our_back_project - sk_image_iradon) ** 2)) / (
+        np.max(sk_image_iradon - np.min(sk_image_iradon))
```

Comment:
I think you're missing a parenthesis for `np.max(...)`.

Reply:
<img width="763" alt="Screenshot 2024-08-06 at 11 30 07 AM" src="https://github.com/user-attachments/assets/9c156a18-a68f-4083-9566-14fe49d595de">
good catch, fixed and resolving for now